### PR TITLE
Dashboard init

### PR DESCRIPTION
**Pull request by QuickFinesse**

### CODE REVIEW

File: web/components/dashboard/drawer/drawerHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth="2"
+                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+              />
+            </svg>
+          </button>
+          <button className="btn btn-ghost btn-circle">
+            <div className="indicator">
+              <svg
```
Comment (zelnaut) [inline web/components/dashboard/drawer/drawerHeader.tsx:49]: Minor, I hate how noisy SVGs are so usually I shove them in a seperate file, don't know if that's 'best practices' or not.

File: web/components/dashboard/drawer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import DrawerHeader from './drawerHeader';
+
+const Drawer = ({ children }: any) => {
```
Comment (zelnaut) [inline web/components/dashboard/drawer/index.tsx:5]: `any` 😠 
Also useless fragment below.

File: web/components/dashboard/table/index.tsx [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+function DebouncedInput({
+  value: initialValue,
+  onChange,
+  debounce = 500,
+  ...props
+}: {
+  value: string | number;
+  onChange: (value: string | number) => void;
+  debounce?: number;
+} & Omit<React.InputHTMLAttributes<HTMLInputElement>, 'onChange'>) {
```
Comment (zelnaut) [inline web/components/dashboard/table/index.tsx:255]: Never seen this syntax before.
Kinda cool
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

File: web/pages/dashboard/jobs.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+const Jobs = () => {
+  const { data, isLoading } = trpc.useQuery(['job.all'], {
+    refetchOnWindowFocus: false,
+  });
+
+  console.log({ data });
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
```
Comment (zelnaut) [inline web/pages/dashboard/jobs.tsx:13]: Guessing we'll want to add a spinner here, or something nicer eventually.

**Review by zelnaut — DISMISSED** [review]
Looks good, go ahead and merge

File: web/pages/dashboard/jobs.tsx (hunk 4 above)
Comment (QuickFinesse) [reply web/pages/dashboard/jobs.tsx:13]: Yup good callout

File: web/components/dashboard/drawer/drawerHeader.tsx (hunk 1 above)
Comment (QuickFinesse) [reply web/components/dashboard/drawer/drawerHeader.tsx:49]: True, good call out we can shove them in `/public` where all our pictures will live

**Pull request merged**